<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Communication Mode :</string>
        </property>
        <property name="buddy">
         <cstring>rd_send</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rd_send">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Click this for sending a file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">cmode_radio</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rd_recv">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Click this for receive a file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Receive</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">cmode_radio</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>File: </string>
        </property>
        <property name="buddy">
         <cstring>txt_fpath</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_fpath"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_load">
        <property name="text">
         <string>Load /Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>IPv4 Send/Receiver:</string>
        </property>
        <property name="buddy">
         <cstring>txt_fsysip</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_fsysip">
        <property name="inputMask">
         <string>000.000.000.000;_</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stegano Mode :</string>
        </property>
        <property name="buddy">
         <cstring>rd_burst</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rd_secure">
        <property name="text">
         <string>Secure</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rd_burst">
        <property name="text">
         <string>Burst</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Password : </string>
        </property>
        <property name="buddy">
         <cstring>txt_passwd</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_passwd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Security Meter :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pg_smeter">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="btn_decrypt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resume Decryption</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Status: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_status">
        <property name="text">
         <string>100KB/s , 1 of4000 packet</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pg_progress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="txt_verbose">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_new">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>New File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_setTimeOut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set TimeOut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>System IPv4 Adress(es) :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbo_sysip">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;This lists all IPv4 addresses of the system&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="cmode_radio"/>
 </buttongroups>
</ui>
